### PR TITLE
A Finnish translation for Blanket

### PR DESCRIPTION
**Pull request by heidiwenger**
Finnish translation file. I'm new here so bare with me. There were no instructions on how to add the translation to the project? So i'm putting this here. I hope i'm not totally off.

### CODE REVIEW

**Comment by rafaelmardojai** [comment]
You also need to add `fi` in `LINGUAS`.

File: po/fi.po [hunk 1]
```diff
@@ -0,0 +1,227 @@
+# SOME DESCRIPTIVE TITLE.
```
Comment (heidiwenger) [inline po/fi.po:1]: Finnish translation for Blanket

**Comment by rafaelmardojai** [comment]
> There were no instructions on how to add the translation to the project?

Yep sorry, I plan to move to Weblate soon, so things will be easier.

File: po/fi.po [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the blanket package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
```
Comment (heidiwenger) [inline po/fi.po:4]: Heidi Wenger <heidi.wenger@protonmail.com>, 2021.

**Comment by heidiwenger** [comment]
All checks have passed? Is this ok now? I had basically no idea what i was doing... Just following instinct :D

**Comment by rafaelmardojai** [comment]
> All checks have passed? Is this ok now? I had basically no idea what i was doing... Just following instinct :D

Well that only means that the PR is mergeable and the Flatpak build didn't fail :P 

You also need to add fi in LINGUAS. But no rush, I will merge and fix anything needed.

**Comment by heidiwenger** [comment]
> > All checks have passed? Is this ok now? I had basically no idea what i was doing... Just following instinct :D
> 
> Well that only means that the PR is mergeable and the Flatpak build didn't fail :P
> 
> You also need to add fi in LINGUAS. But no rush, I will merge and fix anything needed.

Thank you VERY much! This kind of kindness i appreciate <3

**Pull request merged**